applications/adv_networking_bench/adv_networking_bench_default_tx_rx.yaml

```diff
@@ -122,14 +107,14 @@ bench_rx:
   header_size: 64
 
 bench_tx:
-  eth_dst_addr: 48:b0:2d:f4:04:48   # Destination MAC
+  eth_dst_addr: 48:b0:2d:ee:83:ad     # Destination MAC
```

Comment:
Are these mock values or some actual test machine? Looks like this is overwritten for testing. Is there a better or more general way to prompt developers for expected addresses without inlining a real value?

Reply:
This is for my system. I don't need to commit that change, I'll revert this. I initially wanted the test to dynamically set the value but a mock value seemed sufficient. Docs will eventually come to help users fill all this (post testing and packaging work).

@cliffburdick as discussed on slack, even with a garbage value here, the Tx/Rx works. Is that expected?

Reply:
It works because your NIC is in promiscuous mode. If it wasn't then all the packets would be dropped.

Reply:
I updated to a generic value in #611 @tbirdso